docs/root/modules/crowdstrike/schema.md

```diff
@@ -0,0 +1,103 @@
+## Crowdstrike Schema
+
+.. _crowdstrike_schema:
+
+### CrowdstrikeHost
+
+Representation of a Crowdstrike Host
+
+| Field | Description |
+|-------|-------------|
+| firstseen | Timestamp of when a sync job first discovered this node |
+| lastupdated | Timestamp of the last time the node was updated |
+| id | The device ID for this host |
+| cid | The customer ID |
+| instance\_id | The AWS instance ID associated with this host |
+| status | Containment Status of the machine. "Normal" denotes good operations; other values might mean reduced functionality or support. |
+| hostname | The name of the machine. |
+| machine\_domain | Active Directory domain name. |
+| crowdstrike\_first\_seen | Timestamp of device’s first connection to Falcon |
+| crowdstrike\_last\_seen | Timestamp of device’s most recent connection to Falcon |
+| local\_ip | The device's local IP address. |
+| external\_ip | External IP of the device, as seen by CrowdStrike. |
+| cpu\_signature | The CPU signature of the device. |
+| bios\_manufacturer | Bios manufacture name. |
+| bios\_version | Bios version. |
+| mac\_address | The MAC address of the device |
+| os\_version | Operating system version. |
+| os\_build | The build of the OS |
+| platform\_id | CrowdStrike agent configuration notes |
+| platform\_name | Operating system platform. |
+| service\_provider | The service provider for the device. |
+| service\_provider\_account\_id | The service provider account ID associated with this device |
+| agent\_version | CrowdStrike agent configuration notes |
+| system\_manufacturer | Name of system manufacturer |
+| system\_product\_name | Name of system product |
+| product\_type | The product type |
+| product\_type\_desc | Name of product type. |
+| provision\_status | The provision status of the device |
+| reduced\_functionality\_mode | Reduced functionality mode (RFM) status |
+| kernel\_version | Kernel version of the host OS. |
+| major\_version | Major version of the Operating System |
+| minor\_version | Minor version of the Operating System |
+| tags | Grouping tags for the device |
+| modified\_timestamp | The last time that the machine record was updated. Can include status like containment status changes or configuration group changes |
+
+#### Relationships
+
+- CrowdstrikeHost has SpotlightVulnerability
+    ```
+    (CrowdstrikeHost)-[HAS_VULNERABILITY]->(SpotlightVulnerability)
+    ```
+
+### SpotlightVulnerability
```

Comment:
Not familiar with Crowdstrike, what's a SpotlightVulnerability?

More broadly, is it possible to add links to corresponding vendor-provided API docs for each of these resources?

Reply:
I think the docs for crowdstrike are only available if you're paying for the product :(

Crowdstrike is the company, and each of their individual products have names. Hosts are named under crowdstrike, but vulnerabilities are `spotlight`, sensors are under `falcon`, etc.